ts/src/utils/seeds.ts

```diff
@@ -0,0 +1,24 @@
+import { utils } from "../index";
+import { PublicKey } from "@solana/web3.js";
+
+/**
+ * can be used like that:
+ *     const [pda, bump] = await PublicKey.findProgramAddress(
+ *       Seeds.from(signer.publicKey, "input"),
+ *       this.program.programId
+ *     )
+ * or like that:
+ *     const [pda, bump] = await PublicKey.findProgramAddress(
+ *       [signer.publicKey.toBuffer(), ...Seeds.from("input")],
+ *       this.program.programId
+ *     )
+ */
+export function from(...stringSeeds: (string | PublicKey)[]): Buffer[] {
```

Comment:
Let's move this into the publicKey namespace and rename the function to `seeds`.

So we have `anchor.utils.publicKey.seeds()`